plugins/blame.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: blame
+spec:
+  version: v0.0.5
+  homepage: https://github.com/knight42/kubectl-blame
+  shortDescription: Show the manager of each field of given resource with information from managed fields.
```

Comment:
> Show the manager of each field of given resource with information from managed fields.

unfortunately this is too long to fit. Limit to 50 chars if you can. e.g. `Show who edited resource fields` is better.

> with information from managed fields.

"How" you do it is a detail, so it should go in `description`.